### PR TITLE
[NP-10024] BroadcastWebAgent

### PR DESCRIPTION
**Pull request by jlhughes**
BroadcastWebAgent - Proxy request to each endpoint.  Return first success.
DIGBroadcastWebAgentTest - Test case using the DIG Java Client to test the BroadcastWebAgent.

### CODE REVIEW

File: src/foam/nanos/http/BroadcastWebAgent.js [hunk 1]
```diff
@@ -0,0 +1,295 @@
...
+      class: 'StringArray',
+      factory: function() {
+        return ['Accept-Language', 'Content-Type', 'User-Agent'];
+      },
+      javaFactory: `
+        return new String[] {"Accept-Language", "Content-Type", "User-Agent"};
+      `
+    },
+    {
+      documentation: 'Do not wait for a pass or fail on any of hte proxy requests. Immediately return 200',
```
Comment (kgrgreer) [inline src/foam/nanos/http/BroadcastWebAgent.js]: s/hte/the/

File: src/foam/nanos/http/test/DIGBroadcastWebAgentTest.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+      section: 'details'
+    }
+  ],
+
+  methods: [
+    {
+      name: 'runTest',
+      args: 'X x',
+      javaCode: `
+      String url = "http://" + System.getProperty("hostname", "llocalhost") + ":" + System.getProperty("http.port", "8080");
```
Comment (kgrgreer) [inline src/foam/nanos/http/test/DIGBroadcastWebAgentTest.js]: Is that meant to be llocalhost or localhost?

File: src/foam/nanos/http/BroadcastWebAgent.js (hunk 1 above)
Comment (jlhughes) [inline src/foam/nanos/http/BroadcastWebAgent.js]:
```suggestion
      documentation: 'Do not wait for a pass or fail on any of the proxy requests. Immediately return 200',
```

File: src/foam/nanos/http/test/DIGBroadcastWebAgentTest.js (hunk 2 above)
Comment (jlhughes) [inline src/foam/nanos/http/test/DIGBroadcastWebAgentTest.js]:
```suggestion
      String url = "http://" + System.getProperty("hostname", "localhost") + ":" + System.getProperty("http.port", "8080");
```

**Pull request merged**